### ISSUE TITLE
`data` folder in `Dummy_disease_test` is unused

### ISSUE THREAD

**Issue by alexdewar** [imperialCHEPI/healthgps-examples#14]
I previously updated all the config files to use a URL as their data source rather than a data directory, including the config file for `Dummy_disease_test`. This means that the contents of `Dummy_disease_test/data` aren't actually used anywhere.

Is this ok? Do we want to provide an example of how to use a config file in combination with a data directory (as opposed to a URL)? Or should we just delete the directory?

@jamesturner246?

**Comment by alexdewar** [comment]
See also #15

**Comment by jamesturner246** [comment]
Good question. Are we worried about the testsuite failing if ran without network connection? If not, I suppose we can delete it, and make a note in the docs explaining how to instead use a local copy.

**Comment by alexdewar** [comment]
I don't think so. This repo isn't (yet) used in HealthGPS's test suite and doesn't have any tests of its own (but I want to do #7 at some point).

The user doesn't need a local copy of `data` for these examples, but it is something that a user might want to do and is currently undocumented. (The documentation is actually a bit out of date, see https://github.com/imperialCHEPI/healthgps/issues/472 and https://github.com/imperialCHEPI/healthgps/issues/473)
